tests/test_product.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from buy_together_app.models import Product
+
+QR = "Q5o76MbdiNbXprNEnHfpcGWFp1CMF8XY"
+NAME = "Banana"
+DESCRIPTION = "Good!"
+
+
+@pytest.fixture
+def product0():
+    return Product(qr_code=QR, product_name=NAME, description=DESCRIPTION)
+
+
+@pytest.fixture
+def saved_product0(product0):
+    product0.save()
+    return product0
+
+
+test_data = [
+            Product('5yMcE3UrEEmZBukWZBrlCKnu0K5j24UY', 'Apple', 'Sweety!'),
+            Product('Q5o76MbdiNbXprNEnHfpcGWFp1CMF8ad', 'Banana', "Good!"),
```

Comment:
You're coupling the test data to the migration content. If you do that, maybe you can define those values there and then import them here to avoid the duplicate code.

Reply:
Done